website/content/docs/enterprise/license/faq.mdx

```diff
@@ -54,19 +59,30 @@ If you have built automation around `nomad license put`, this command will fail
 
 ## Q: What are some other gotchas?
 
-You might not have access to your organization's Nomad license. 
+You might not have access to your organization's Nomad license.
 The  6-hour temporary license window is now removed, and a license is required for proper boot-up.
 Running `nomad agent -dev` without a license will fail.
 
 
 ## Q: How can a user get their existing license to put on disk for an upgrade?
 
 If you are an existing HashiCorp enterprise customer you may contact your organization's customer success manager (CSM) or email support-softwaredelivery@hashicorp.com for information on how to get your organization's enterprise license.
-You can use `nomad license get` to retrieve information about the license, but not the license itself. 
+You can use `nomad license get` to retrieve information about the license, but not the license itself.
 
-## Q: What happens when a license expires?
-If a license expires after a server node joins the cluster, the existing Enterprise features will continue to operate until a grace period expires. 
-The grace period duration is 1 day, and upon the end of the grace period the Nomad servers will shut down. 
+## Q: What happens when a license expires or terminates?
+
+As a Nomad Enterprise license approaches its expiration time, Nomad servers will
+periodically log a warning message about the approaching expiration. The grace
+period is defined as the time between license expiration and license
+termination. (This is typically one day for evaluation licenses, and much longer
+for non-evaluation licenses.) Customers must provide a valid license before the
```

Comment:
The Vault docs from pre-1.12 say 10 years, but I don't know for sure that this is what was actually issued to Nomad customers because I don't have access to that data. Rather than promising anything in particular, I figure this language will send folks to their sales person.